benchmark/string_decoder/string-decoder.js

```diff
@@ -9,7 +9,7 @@ var bench = common.createBenchmark(main, {
   n: [25e4]
 });
 
-var UTF_ALPHA = 'Bl�b�rsyltet�y';
+var UTF_ALPHA = 'Blåbærsyltetøy';
```

Comment:
Appears to have been caused by my text editor being overly "helpful".  The change does not appear to have any effect on the actual benchmark so I'm inclined to leave it as it for now.